scripts/configure.js

```diff
@@ -323,6 +323,10 @@ const getConfig = (() => {
             "metro.config.js": {
               source: path.join(testAppPath, "example", "metro.config.js"),
             },
+            ".gitignore": fs
+              .readFileSync(path.join(process.cwd(), "example", ".gitignore"))
+               + "\n" + 
```

Comment:
I'm not sure you can just concatenate the two files. `windows/.gitignore` contains rules that may exclude names that are perfectly fine to have under the project folder. Can we instead copy the file under the Windows configuration section instead?